### ISSUE TITLE
Perplexity and citations

### ISSUE THREAD

**Issue by rdiaz02** [rksm/org-ai#135]
In the README, https://github.com/rksm/org-ai?tab=readme-ov-file#perplexityai, it says "Note: Currently the perplexity.ai does not give access to references/links via the API so Emacs will not be able to display references. They have a beta program for that running and I sure hope that this will be available generally soon."

As from a few weeks ago, the citations are being returned as part of the output; see, for example, https://github.com/karthink/gptel/issues/299  (and I just verified with gptel and using log-level to debug, and I do find the citations in the log).

**Comment by metachip** [comment]
Yep, thanks for this! My comment on this: https://github.com/karthink/gptel/issues/299#issuecomment-2566826441

**Comment by rksm** [comment]
Awesome, I'll be adding support for this!
